From 914cc2a5576cd45afa5f6d6f5327da9cc4d3bbac Mon Sep 17 00:00:00 2001
Message-Id: <914cc2a5576cd45afa5f6d6f5327da9cc4d3bbac.1387276076.git.minovotn@redhat.com>
In-Reply-To: <e610718166120379517e80d1a7aa12d60294209b.1387276076.git.minovotn@redhat.com>
References: <e610718166120379517e80d1a7aa12d60294209b.1387276076.git.minovotn@redhat.com>
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Tue, 10 Dec 2013 14:33:38 +0100
Subject: [PATCH 08/16] block: Dont ignore previously set bdrv_flags

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1386686018-11942-1-git-send-email-pbonzini@redhat.com>
Patchwork-id: 56109
O-Subject: [RHEL 7.0 qemu-kvm PATCH] block: Dont ignore previously set bdrv_flags
Bugzilla: 1039501
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>

From: "M. Mohan Kumar" <mohan@in.ibm.com>

Bugzilla: 1039501

Brew build: 6700417

bdrv_flags is set by bdrv_parse_discard_flags(), but later it is reset
to zero.

Signed-off-by: M. Mohan Kumar <mohan@in.ibm.com>
Message-id: 1376483201-13466-1-git-send-email-mohan@in.ibm.com
Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
(cherry picked from commit 8b7a5415f9297c1fbae5bff28dcb059d50c1b045)
---
 blockdev.c | 1 -
 1 file changed, 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 blockdev.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/blockdev.c b/blockdev.c
index a9c5d32..cdd453a 100644
--- a/blockdev.c
+++ b/blockdev.c
@@ -384,7 +384,6 @@ static DriveInfo *blockdev_init(QDict *bs_opts,
         }
     }
 
-    bdrv_flags = 0;
     if (qemu_opt_get_bool(opts, "cache.writeback", true)) {
         bdrv_flags |= BDRV_O_CACHE_WB;
     }
-- 
1.7.11.7

